python/hsfs/client/external.py

```diff
@@ -115,6 +117,29 @@ def __init__(
                 "spark.hadoop.hops.ssl.keystore.name"
             )
 
+    def validate_spark_configuration(self, _spark_session):
+        exception_text = "Spark is misconfigured for communication with Hopsworks, missing or invalid property: "
+
+        configuration_dict = {"spark.hadoop.hops.ssl.trustore.name": None,
+                "spark.hadoop.hops.rpc.socket.factory.class.default": "io.hops.hadoop.shaded.org.apache.hadoop.net.HopsSSLSocketFactory",
+                "spark.serializer": "org.apache.spark.serializer.KryoSerializer",
+                "spark.hadoop.hops.ssl.hostname.verifier": "ALLOW_ALL",
+                "spark.hadoop.hops.ssl.keystore.name": None,
+                "spark.hadoop.fs.hopsfs.impl": "io.hops.hopsfs.client.HopsFileSystem",
+                "spark.hadoop.hops.ssl.keystores.passwd.name": None,
+                "spark.hadoop.hops.ipc.server.ssl.enabled": "true",
+                "spark.sql.hive.metastore.jars": None,
+                "spark.hadoop.client.rpc.ssl.enabled.protocol": "TLSv1.2",
+                "spark.hadoop.hive.metastore.uris": None}
```

Comment:
This looks weirdly formatted - did you use `black`?

Reply:
Sorry i don't understand what you mean here.